examples/README.md

```diff
@@ -455,4 +455,29 @@ Device name is: AMD Radeon PRO W6800
 Running with single precision real data type:
 DFT example run_time dispatch
 DFT example ran OK
-```
\ No newline at end of file
+```
+
+```none
+$ LD_LIBRARY_PATH=lib ./bin/example_dft_real_fwd_usm 
+########################################################################
+# DFTI complex in-place forward transform with USM API example:
+#
+# Using APIs:
+#   USM forward complex in-place
+#   Run-time dispatch
+#
+# Using single precision (float) data type
+#
+# Device will be selected during runtime.
+# The environment variable SYCL_DEVICE_FILTER can be used to specify
+# SYCL device
+#
+########################################################################
+
+Running DFT complex forward example on GPU device
+Device name is: Intel(R) UHD Graphics 750
+Running with single precision real data type:
+DFT example run_time dispatch
+Caught std::exception:
+	oneMKL: dft/backends/portfft/commit: function is not implemented portFFT only supports complex to complex transforms
```

Comment:
I think we should change the example so something that actually does something:
* Demoing unsupported functionality isn't useful.
* We'll probably forget to update this when the real to complex work is merged.

Reply:
We discussed this here: https://github.com/FMarno/oneMKL/pull/1#discussion_r1387853267

Do you have a suggestion for what to do?
I don't really want to add a new example. I could potentially not enable the example for portFFT.